data/json/recipes/basecamps/recipe_modular_saltworks/recipe_modular_saltworks_wad.json

```diff
@@ -0,0 +1,98 @@
+    "never_learn": true,
+    "construction_blueprint": "fbmsw_saltpan_wad_northeast",
+    "blueprint_name": "northeast saltpan",
+    "blueprint_requires": [ { "id": "fbmsw_0" } ],
+    "blueprint_provides": [ { "id": "fbmsw_northeast" }, { "id": "Salt_Pan" } ],
+    "blueprint_excludes": [ { "id": "fbmsw_northeast" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_shack_wood_southeast",
+    "description": "We could use a storage room for chemicals and tools, so build an adobe shack with a roof on the southeast side of the saltworks field.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmsw_shack_wad_southeast",
+    "blueprint_name": "southeast storage room",
+    "blueprint_requires": [ { "id": "fbmsw_0" } ],
+    "blueprint_provides": [ { "id": "fbmsw_southeast" } ],
+    "blueprint_excludes": [ { "id": "fbmsw_southeast" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_brewery0_wad_southwest",
+    "description": "We could take advantage of the the designation of this area to build an adobe brewery.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmsw_brewery0_wad_southwest",
+    "blueprint_name": "southwest brewery",
+    "blueprint_requires": [ { "id": "faction_base_saltworks_0" } ],
+    "blueprint_provides": [ { "id": "fbmsw_southwest" }, { "id": "brewery" } ],
+    "blueprint_excludes": [ { "id": "fbmsw_southwest" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_brewery1_wad_southwest",
+    "description": "We could uprade the workspace in the brewery to add a still, wood stove and work table.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmsw_brewery1_wad_southwest",
+    "blueprint_name": "southwestwest brewery1",
+    "blueprint_requires": [ { "id": "fbmsw_southwest" } ],
+    "blueprint_provides": [ { "id": "fbmsw_southwest" } ],
+    "blueprint_excludes": [ { "id": "fbmsw_southwest" } ],
+    "blueprint_autocalc": true
+  },
+  {
+    "type": "recipe",
+    "result": "faction_base_modular_brewer2_wad_southwest",
+    "description": "We could use fill out the space with extra kegs and storage racks.",
+    "category": "CC_BUILDING",
+    "subcategory": "CSC_BUILDING_BASES",
+    "autolearn": false,
+    "never_learn": true,
+    "construction_blueprint": "fbmsw_brewery2_wad_southwest",
+    "blueprint_name": "southwest brewery2",
+    "blueprint_requires": [ { "id": "fbmsw_southwest" } ],
+    "blueprint_provides": [ { "id": "fbmsw_southwest" } ],
+    "blueprint_excludes": [ { "id": "fbmsw_souththwest" } ],
```

Comment:
typo in blueprint excludes southtwest

Reply:
Ok I think I see what I needed to do there now.  I've made changes and will push. I've renamed the upgrades to fbms...1 and 2.